.circleci/config.yml

```diff
@@ -21,7 +21,10 @@ reference:
 defaults: &defaults
   working_directory: ~/app
   docker:
-    - image: celohq/node10-gcloud:v3
+    - image: gcr.io/$GOOGLE_PROJECT_ID/circleci:latest
```

Comment:
Hey @yorhodes rather than using `latest` I think a specific version would be better here, because if a new `circleci` image is pushed that is not backwards compatible all the builds will break.

Also where does the $GOOGLE_PROJECT_ID get injected into the build?

Reply:
the env variables are configured as circleci secrets